about.html

```diff
@@ -0,0 +1,137 @@
+      href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css"
+    />
+    <!-- font awesome -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css"
+      integrity="sha512-YWzhKL2whUzgiheMoBFwW8CKV4qpHQAEuvilg9FAn5VJUDwKZZxkJNuGM4XkWuk94WCrrwslk8yWNGmY1EduTA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="style/about.css" />
+    <title>About ProCoder</title>
+  </head>
+  <body>
+    <header>
+      <section>
+        <div class="black-nav h-100 d-none d-md-flex align-items-center justify-content-end gap-3 pe-5">
+          <a href="#"><i class="fab fa-facebook-f"></i></a>
+          <a href="#"><i class="fab fa-twitter"></i></a>
+          <a href="#">Logout</a>
+          <a href="#">My Page</a>
+          <a href="#">English</a>
+        </div>
+      </section>
+      <nav class="navbar navbar-expand-md navbar-light">
+        <div class="container-fluid">
+          <a class="navbar-brand d-none d-md-inline" href="index.html">
+            <img class="nav-img" src="images/logo1.jpg" alt="">
+          </a>
+          <button
+            class="navbar-toggler border-0 fs-1 fw-bold"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarSupportedContent"
+            aria-controls="navbarSupportedContent"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <i class="fa-solid fa-bars"></i>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+              <li class="ms-auto d-md-none">
+                <i class="fa-solid fa-x"></i>
+              </li>
+              <li class="nav-item p-2">
+                <a class="nav-a active-nav" href="index.html">Home</a>
+              </li>
+              <li class="nav-item p-2">
+                <a class="nav-a" href="about.html">About</a>
+              </li>
+              <li class="nav-item p-2">
+                <a class="nav-a join-us" href="#">Join Us</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+
+      <section id="main-header">
+        <h3 class="main-header-h3 d-md-none">"Hello! Sharing world"</h3>
+        <h1 class="main-header-h1">ProCoder Full-Stack Diploma 2021</h1>
+        <p class="main-header-desc">The Global Diploma brings together the community of experts, academics, and activites who comprise the ProCoder affiliate network in a different countery every two. This year, we hope to expand our invitation list including organizations and individuals</p>
+        <p class="main-header-p">Please contact us per Email for any further questions about ProCoder Diploma 2021!</p>
+        <p class="main-header-mail">ProCoder@gmail.com</p>
+      </section>
```

Comment:
- [ ] This `section` (the `hero` section) shouldn't belong to the `header` tag, please move it to your main content.

Reply:
Done